### DISCUSSION TITLE
Unable to send emails.

### DISCUSSION THREAD

**Discussion by Salman9000** [pocketbase/pocketbase#5110] [Q&A]
Hello. After reading through the docs and going through the discussions and issues, i am still unable to send emails. Even test emails not going through. Following are my screenshots of how I am sending it.

I created an account on mailtrap.io
I used these credentials. ![image](https://github.com/pocketbase/pocketbase/assets/26120951/3733aeab-7153-4ee0-87a2-02a7bddf1c1c)

Then in the pocketbase admin UI, settings -> mail settings.
Added the config 
![image](https://github.com/pocketbase/pocketbase/assets/26120951/9754ec08-05bd-4736-9002-2fed5945b7e4)

Sent a test email to my own gmail 
![image](https://github.com/pocketbase/pocketbase/assets/26120951/976d9405-bf06-42a9-b7f1-e2d5b8008b29)

After more than 15 seconds i get this timeout error 
![image](https://github.com/pocketbase/pocketbase/assets/26120951/e4b32351-e535-4cc1-a5d8-713283797201)

Logs: 
![image](https://github.com/pocketbase/pocketbase/assets/26120951/c508582e-71fb-41cf-a9a2-5caaf800a3dc)

**Comment by ganigeorgiev** [comment]
It works correctly for me when testing with 587 port.

Additionally, if you are not testing locally, note that many VPS providers require to contact them to allow outbound requests through ports 25, 465, etc. as some kind of spam/ip-reputation protection.

**Reply by Salman9000** [reply]
I am using coolify to host pocketbase. 
I used other ports but they are also not working

**Reply by ganigeorgiev** [reply]
Which hosting/VPS provider?

**Reply by ganigeorgiev** [reply]
If you are testing locally then it is possible to be something related to the Coolify configuration, I'm not sure. To rule out Coolify issue try to test with just the the PocketBase executable (aka. `./pocketbase serve`).
